lib/Viroverse/Search/Faceted.pm

```diff
@@ -237,6 +237,13 @@ often have their own bespoke UI handling (and thus the UI can set appropriate
 labels), facets more naturally share UI handling and doing so is facilitated by
 providing a label with the data.
 
+=item join
```

Comment:
The `join` feature is unnecessary after switching to using a materialized view; go ahead and revert the commit introducing it.